src/pages/collection/series/SeriesTags.tsx

```diff
@@ -74,7 +74,7 @@ const SeriesTags = () => {
           </div>
         </div>
         <div className="grid grid-cols-3 gap-8">
-          {map(tags, item => <SeriesTag item={item} />)}
+          {map(tags, (item, index) => <SeriesTag key={index} item={item} />)}
```

Comment:
It is generally advised to use the actual id if it is available rather than an array index. Check the pitfall note here https://react.dev/learn/rendering-lists#where-to-get-your-key

Reply:
Fixed.